src/styles/Landing.styled.ts

```diff
@@ -40,6 +40,7 @@ export const Occupation = styled.h3`
 
 export const Container = styled.div`
   margin-top: 10%;
+  overflow: hidden;
```

Comment:
The other boxes for payment options shouldn't be hidden. Try changing the flex direction so that you have the three boxes in a vertically scrollable way. Also, try to still have the `Most Popular` option at the top (you can order with flex [order](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items))!